Felix45/Questioner
# Ch set up project structure 162971901

**What does this pull request do**
- Sets up the directory and file structure for the Questioner App

**Description of the task to be completed**
Sets up the directory and file structure for the Questioner application

**How should this be manually tested**

- Clone or fork the repository 
- Open a terminal
- Change your working directory to the project directory
- Checkout to set-up-project-structure-162971901
- To see the directory structure  open the UI direcrory as a workspace on your editor of choice
![project-structure](https://user-images.githubusercontent.com/16392046/50707722-e07b3d80-1072-11e9-9bf3-6b5e932de1ea.png)

**Relevant pivotal tracker stories**
https://www.pivotaltracker.com/n/projects/2235209
